.coveragerc

```diff
@@ -574,7 +574,68 @@ omit =
     homeassistant/components/tapsaff/binary_sensor.py
     homeassistant/components/tautulli/sensor.py
     homeassistant/components/ted5000/sensor.py
-    homeassistant/components/telegram/notify.py
+    homeassistant/components/temper/sensor.py
```

Comment:
These unrelated changes should not be here.

Reply:
Resolved. It looks clean now.